internal/service/ocm/ocm_interface.go

```diff
@@ -0,0 +1,34 @@
+package ocm
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/redhatinsights/mbop/internal/config"
+	"github.com/redhatinsights/mbop/internal/models"
+)
+
+type OCM interface {
+	InitSdkConnection(ctx context.Context) error
+	CloseSdkConnection()
+	GetUsers(users models.UserBody, q models.UserQuery) (models.Users, error)
+	GetOrgAdmin([]models.User) (models.OrgAdminResponse, error)
+}
```

Comment:
this is probably a silly question - but how would i convert a token to an orgID? Would i call get users with the token? would that make sense to add to the interface? 

only reason i ask is i'll need to do that for the registrations endpoint.

Reply:
For registrations yes, you'll probably need to call this endpoint with the username from the token to get back the org.